x-pack/plugins/canvas/__tests__/fixtures/workpads.ts

```diff
@@ -53,6 +53,7 @@ const BaseElement: CanvasElement = {
 export const workpads: CanvasWorkpad[] = [
   {
     ...BaseWorkpad,
+    fromTemplate: 'APM Template',
```

Comment:
Ok, so this preps us for APM Template, but do the names have to be hardcoded here, or can they just take the name of the actual template as a variable?

Reply:
Both this and the "SIEM Template" string in your other comment are just for our unit tests! This workpad isn't used anywhere other than our test cases